src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const gallery = document.querySelector('#thumbs');
+
+const handleClick = (e) => {
+  e.preventDefault();
+
+  const target = e.target.closest('.list-item__link');
+
+  if (target.nodeName !== 'A') {
```

Comment:
The condition `target.nodeName !== 'A'` might not work as expected because `nodeName` returns the tag name in uppercase. Consider using `target.tagName !== 'A'` or `target.nodeName.toUpperCase() !== 'A'` to ensure the comparison is case-insensitive.